documentation/configure.markdown

```diff
@@ -274,6 +274,8 @@ Do not use the `username` or `password` settings.
 
 This controls how often, in minutes, to compact the database. The compaction process reclaims space and deletes unnecessary rows. If not supplied, the default is every 60 minutes.
 
+If `gc-interval` is set to 0, all GC processes will be disabled. This may be useful, for example, for a read-only PuppetDB instance using a replicated database. It should not be used for a PuppetDB instance that will receive any commands. 
```

Comment:
Seems like a more accurate way to put this is, if running more than one PuppetDB instance sharing a single database, only 1 of those instances should have GC enabled. Read vs. write vs. commands doesn't matter.